frontend/components/LoginForm/LoginForm.tsx

```diff
@@ -50,7 +54,7 @@ const LoginForm: React.FC = () => {
               },
             })}
           />
-          <ErrorMessage errors={errors} name="email" as="p" />
+          <ErrorMessage errors={errors} name="email" as="em" />
```

Comment:
Semantically, we'd want to keep the `p` tag and use `font-style: italic`

Reply:
Isn't using `em` and `strong` considered to be the correct approach for semantic markup and also for screen readers?

Reply:
In this case it's more of a design effect and "style" than actually emphasizing text. `em` and `strong` as semantic elements are generally reserved for linguistic usage. From the [HTML spec doc](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-em-element):

> The placement of stress emphasis changes the meaning of the sentence. The element thus forms an integral part of the content. The precise way in which stress is used in this way depends on the language.

Reply:
Okie dokie changed 😄